README.md

```diff
@@ -1 +1,71 @@
-# eip-draft
\ No newline at end of file
+# EIP-XXXX: Cryptographic Proof of Exchange Solvency
+
+## Preamble
+
+```
+EIP: XXXX
+Title: Cryptographic Proof of Exchange Solvency
+Author: [ADD NAMES HERE]
+Type: Standards Track (ERC)
+Category: ERC
+Status: Draft
+Created: 2023-08-30
+```
+
+## Abstract
+
+This EIP proposes a standard for centralized exchanges to cryptographically prove their solvency without revealing confidential information about user balances using cryptographic techniques like zero-knowledge proofs.
```

Comment:
Privacy relates to: 
- individual user balances
- aggregated information related to user balances